geopy/geocoders/mapbox.py

```diff
@@ -0,0 +1,215 @@
+from geopy.compat import urlencode, quote, string_compare
+from geopy.exc import (
+    GeocoderQueryError,
+)
+from geopy.geocoders.base import DEFAULT_SENTINEL, Geocoder
+from geopy.location import Location
+from geopy.point import Point
+from geopy.util import logger
+
+__all__ = ("MapBox", )
+
+
+class MapBox(Geocoder):
+    """Geocoder using the Mapbox API.
+
+    Documentation at:
+        https://www.mapbox.com/api-documentation/
+
+    .. versionadded:: 1.17.0
+    """
+
+    api_path = '/geocoding/v5/mapbox.places/%(query)s.json/'
+
+    def __init__(
+            self,
+            api_key,
+            format_string=None,
+            scheme=None,
+            timeout=DEFAULT_SENTINEL,
+            proxies=DEFAULT_SENTINEL,
+            user_agent=None,
+            ssl_context=DEFAULT_SENTINEL,
+            domain='api.mapbox.com',
+    ):
+        """
+        :param str api_key: The API key required by Mapbox to perform
+            geocoding requests. API keys are managed through Mapox's account
+            page (https://www.mapbox.com/account/access-tokens).
+
+        :param str format_string:
+            See :attr:`geopy.geocoders.options.default_format_string`.
+
+        :param str scheme:
+            See :attr:`geopy.geocoders.options.default_scheme`.
+
+        :param int timeout:
+            See :attr:`geopy.geocoders.options.default_timeout`.
+
+        :param dict proxies:
+            See :attr:`geopy.geocoders.options.default_proxies`.
+
+        :param str user_agent:
+            See :attr:`geopy.geocoders.options.default_user_agent`.
+
+        :type ssl_context: :class:`ssl.SSLContext`
+            :param ssl_context:
+                See :attr:`geopy.geocoders.options.default_ssl_context`.
```

Comment:
Indentation is off by 1 here and the line above. I suppose this would be an error when generating docs. 

These lines should be aligned like that:
https://github.com/geopy/geopy/blob/0a3a60bd6f9cb570e346ffef7953babf630ae8a3/geopy/geocoders/tomtom.py#L50-L52